modules/jaeger-sidecar-automatic.adoc

```diff
@@ -32,7 +32,6 @@ spec:
     spec:
       containers:
       - name: myapp
-        image: acme/myapp:myversion
```

Comment:
This image can remain as it is referring to the app that is having the sidecar injected.